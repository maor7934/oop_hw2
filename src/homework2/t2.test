#check sorting of nodes and children:

CreateNode a 10
CreateNode b 10
CreateNode c 10
CreateNode aba 10
CreateNode z 10
CreateNode az 10
CreateNode cab 10
CreateNode ca 10
CreateNode cac 10

CreateNode zzz 500

CreateGraph A
AddNode A az
AddNode A cab
AddNode A ca
AddNode A z
AddNode A b
AddNode A c
AddNode A aba
AddNode A cac
AddNode A zzz
AddNode A a


AddEdge A z a
AddEdge A z ZZZ
AddEdge A z cac
AddEdge A z aba
AddEdge A z b
AddEdge A z ca
AddEdge A z c


ListNodes A
ListChildren A z
ListChildren A a







