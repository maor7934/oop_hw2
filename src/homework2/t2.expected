#check sorting of nodes and children:

created node a with cost 10
created node b with cost 10
created node c with cost 10
created node aba with cost 10
created node z with cost 10
created node az with cost 10
created node cab with cost 10
created node ca with cost 10
created node cac with cost 10

created node zzz with cost 500

created graph A
added node az to A
added node cab to A
added node ca to A
added node z to A
added node b to A
added node c to A
added node aba to A
added node cac to A
added node zzz to A
added node a to A


added edge from z to a in A
added edge from z to cac in A
added edge from z to aba in A
added edge from z to b in A
added edge from z to ca in A
added edge from z to c in A


A contains: a aba az b c ca cab cac z zzz
the children of z in A are: a aba b c ca cac
the children of a in A are:







